tikv-client/src/main/java/com/pingcap/tikv/PDClient.java

```diff
@@ -42,33 +44,75 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.tikv.kvproto.Metapb.Store;
 import org.tikv.kvproto.PDGrpc;
 import org.tikv.kvproto.PDGrpc.PDBlockingStub;
 import org.tikv.kvproto.PDGrpc.PDStub;
-import org.tikv.kvproto.Pdpb.GetMembersRequest;
-import org.tikv.kvproto.Pdpb.GetMembersResponse;
-import org.tikv.kvproto.Pdpb.GetRegionByIDRequest;
-import org.tikv.kvproto.Pdpb.GetRegionRequest;
-import org.tikv.kvproto.Pdpb.GetRegionResponse;
-import org.tikv.kvproto.Pdpb.GetStoreRequest;
-import org.tikv.kvproto.Pdpb.GetStoreResponse;
-import org.tikv.kvproto.Pdpb.RequestHeader;
-import org.tikv.kvproto.Pdpb.ScatterRegionRequest;
-import org.tikv.kvproto.Pdpb.ScatterRegionResponse;
-import org.tikv.kvproto.Pdpb.Timestamp;
-import org.tikv.kvproto.Pdpb.TsoRequest;
-import org.tikv.kvproto.Pdpb.TsoResponse;
+import org.tikv.kvproto.Pdpb.*;
+import org.tikv.kvproto.Pdpb.Error;
 
 public class PDClient extends AbstractGRPCClient<PDBlockingStub, PDStub>
     implements ReadOnlyPDClient {
-
+  private final Logger logger = LoggerFactory.getLogger(PDClient.class);
   private RequestHeader header;
   private TsoRequest tsoReq;
   private volatile LeaderWrapper leaderWrapper;
   private ScheduledExecutorService service;
   private List<URI> pdAddrs;
 
+  private GetOperatorResponse getOperator(long regionId) {
```

Comment:
pls add some notes to let others know what this function does

Reply:
cause `METHOD_GET_OPERATOR` does not explain itself.

Reply:
`METHOD_GET_OPERATOR` means get pd's operator associated with specific region.

Reply:
cool